client/src/includes/userbar.ts

```diff
@@ -435,12 +444,64 @@ export class Userbar extends HTMLElement {
             currentA11ySelector.addEventListener('click', () => {
               const inaccessibleElement =
                 document.querySelector<HTMLElement>(selectorName);
-              if (!inaccessibleElement) return;
+              const a11yOutlineContainer =
+                this.shadowRoot?.querySelector<HTMLElement>(
+                  '[data-a11y-result-outline-container]',
+                );
+              if (a11yOutlineContainer?.firstElementChild) {
+                a11yOutlineContainer.removeChild(
+                  a11yOutlineContainer.firstElementChild,
+                );
+              }
+              a11yOutlineContainer?.appendChild(
+                a11yOutlineTemplate.content.cloneNode(true),
+              );
+              const currentA11yOutline =
+                this.shadowRoot?.querySelector<HTMLElement>(
+                  '[data-a11y-result-outline]',
+                );
+              if (
+                !this.shadowRoot ||
+                !inaccessibleElement ||
+                !currentA11yOutline ||
+                !a11yOutlineContainer
+              )
+                return;
+
+              const styleA11yOutline = () => {
+                const rect = inaccessibleElement.getBoundingClientRect();
+                currentA11yOutline.style.cssText = `
+                top: ${
+                  rect.height < 5
+                    ? `${rect.top + window.scrollY - 2.5}px`
+                    : `${rect.top + window.scrollY}px`
+                };
+                left: ${
+                  rect.width < 5
+                    ? `${rect.left + window.scrollX - 2.5}px`
+                    : `${rect.left + window.scrollX}px`
+                };
+                width: ${rect.width < 5 ? '5px' : `${rect.width}px`};
+                height: ${rect.height < 5 ? '5px' : `${rect.height}px`};
```

Comment:
This would be a bit neater as:

```suggestion
                width: ${Math.max(rect.width, 5)}px;
                height: ${Math.max(rect.height, 5)}px;
```